python/graphstorm/model/rgcn_encoder.py

```diff
@@ -228,7 +228,8 @@ def forward(self, g, inputs):
 
         def _apply(ntype, h):
             if self.self_loop:
-                h = h + th.matmul(inputs_dst[ntype], self.loop_weight)
+                h = h + inputs_dst[ntype]
```

Comment:
This actually change the model implementation.

Reply:
Revert to the origin. Let's have another PR to fix this.